tutorials/focalboard-project-management/index.mdx

```diff
@@ -0,0 +1,219 @@
+    }
+    }
+    ```
+2. Test the new configuration, enable the site, and reload Nginx to activate the Focalboard site.
+    ```
+    nginx -t
+    ln -s /etc/nginx/sites-available/focalboard.example.com.conf /etc/nginx/sites-enabled/focalboard.example.com.conf
+    service nginx reload
+    ```
+3. Run `certbot` to obtain a TLS certificate for your domain name and let the application configure Nginx automatically.
+    ```
+    certbot
+    ```
+    <Message type="tip">
+      For more details on using [Certbot](https://certbot.eff.org/), refer to our [dedicated documentation](/tutorials/nginx-reverse-proxy/#adding-tls-to-your-nginx-reverse-proxy-using-let's-encrypt).
+    </Message>
+
+## MariaDB configuration
+
+Focalboard uses a MariaDB database to store it's data. Initialize the database engine, create a table and user for Focalboard.
+
+1. Run the following command to initialize the [MariaDB](/tutorials/mariadb-ubuntu-bionic/) database server.
+    ```
+    mysql_secure_installation
+    ```
+    Follow the steps in the wizard:
+    * Enter current password for root: Press **Enter** to continue.
+    * Set root password? **Y**
+    * Create and confirm a new password
+    * Remove anonymous users? **Y**
+    * Disallow root login remotely? **Y**
+    * Remove test database and access to it? **Y**
+    * Reload privilege tables now? **Y**
+2. Log into the MariaDB shell using the MySQL user `root` and the password created in the previous step.
+    ```
+    mysql -u root -p
+    ```
+3. Create a table for Focalboard and grant permission to a MariaDB user. Change the values for `focalboarduser` and `focalboardpassword` to your own user/password combination.
+    ```
+    CREATE DATABASE focalboard;
+    GRANT ALL on focalboard.* to 'focalboarduser'@'localhost' identified by 'focalboardpassword';
+    ```
+    Then leave the MariaDB shell by typing `exit;`.
+4. Open the Focalboard configuration file in a text editor.
+    ```
+    nano /opt/focalboard/config.json
+    ```
+    Edit the database configuration as shown below:
+    ```
+    "dbtype": "mysql",
+    "dbconfig": "focalboarduser:focalboardpassword@tcp(127.0.0.1:3306)/focalboard",
+    ```
+
+## Configure a Focalboard service
+
+We configure a service that will will keep the Focalboard server running across reboots.
+
+1. Create a service description. 
+    ```
+    nano /lib/systemd/system/focalboard.service
+    ```
+    Copy the following content into the file, save it and exit the text editor.
+2. Reload the systemd configuration, enable and start the Focalboard service.
+    ```
+    systemctl daemon-reload
+    systemctl start focalboard.service
+    systemctl enable focalboard.service
+    ```
+
+## Firewall configuration
+
+To avoid the Focalboard application being directly reachable via plain HTTP on Port 8000, we configure [a firewall](/tutorials/installation-uncomplicated-firewall/) to block all incoming ports, except HTTP, HTTPS, and SSH.
```

Comment:
```suggestion
To avoid the Focalboard application being directly reachable via plain HTTP on port `8000`, we configure [a firewall](/tutorials/installation-uncomplicated-firewall/) to block all incoming ports, except HTTP, HTTPS, and SSH.
```